Aikido.Zen.Core/Models/Ip/Blocklist.cs

```diff
@@ -155,6 +183,34 @@ public bool IsIPAllowed(string ip, string endpoint)
             }
         }
 
+        public bool IsIPAllowed(string ip)
+        {
+            _lock.EnterReadLock();
+            try
+            {
+                if (!IPHelper.IsValidIp(ip))
+                {
+                    return true; // Invalid IPs are by default allowed
+                }
+
+                if (_allowedSubnets.HasItems)
```

Comment:
Also yeah would check this before we check if it's a valid IP fyi, makes more sense